frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js

```diff
@@ -180,7 +183,7 @@ const getExplanationForDisabledVolume = (
     return "Volume annotation is disabled while the merger mode is active.";
   }
 
-  if (!isLabelingPossible) {
+  if (!isSegmentationVisibleForMag) {
```

Comment:
One thing I noticed during testing is that this boolean is also false if the segmentation layer is simply disabled (for example, using `3`). Could you adapt the message accordingly?

Reply:
Good catch! Since the existing logic around the disabled tool tooltip (i.e., which text to show) was a bit convoluted, I refactored the corresponding code which made it easier to integrate the case you mentioned. Also, the automatic switching of the tool (if the current one gets disabled), got a bit DRYer due to the refactoring.
Feel free to have another look!